app.py

```diff
@@ -63,36 +63,30 @@ async def on_voice_state_update(member, before, after):
                 await before.channel.edit(name="General")  # Reset channel name
 
 
+session_roles = {782273117321166888, 782423045578948618, 782423808271056917, 853135207862370324, 853135509488009216, 853135836597583902, 931662314074169364, 921725505051459664, 921725841677901905, 698285991110246441}
```

Comment:
These 2 variables have to be changed to lists

Reply:
No, they need to be sets for `intersection` to work.

Reply:
I think a set was defined here, which is fine.

Reply:
Oh ok nevermind then